tests/bootstrap_phpunit.php

```diff
@@ -4,67 +4,4 @@
 
 require __DIR__ . '/bootstrap_common.php';
 
-$phpunitVersionParts = class_exists('\PHPUnit\Runner\Version')
-    ? explode('.', \PHPUnit\Runner\Version::id())
-    : explode('.', PHPUnit_Runner_Version::id());
-define('PHPUNIT_MAJOR', intval($phpunitVersionParts[0]));
-
-if (PHPUNIT_MAJOR >= 8) {
-    require __DIR__ . '/Common/MultiPHPUnitVersionAdapter_typed.php';
-} else {
-    require __DIR__ . '/Common/MultiPHPUnitVersionAdapter_untyped.php';
-}
-
 require_once __DIR__ . '/Appsec/Mock.php';
-
-function update_test_agent_session_token($token) {
-    if (defined('GLOBAL_AUTO_PREPEND_RSRC')) {
-        ini_set("datadog.trace.agent_test_session_token", $token);
-        ftruncate(GLOBAL_AUTO_PREPEND_RSRC, 0);
-        fseek(GLOBAL_AUTO_PREPEND_RSRC, 0);
-        fwrite(GLOBAL_AUTO_PREPEND_RSRC, "<?php " . (getenv('PHPUNIT_COVERAGE') ? " require '" . __DIR__ . "/save_code_coverage.php';" : ""). " ini_set('datadog.trace.agent_test_session_token', '$token');");
-    }
-}
```

Comment:
Looking at the [CI failures](https://app.circleci.com/pipelines/github/DataDog/dd-trace-php/17599/workflows/9aec4845-c3e4-4aba-8079-12f8fd94877d/jobs/5224195?invite=true#step-149-18236_73), it would appear that moving this function from `bootstrap_phpunit` to `bootstrap_common` introduced some errors in our test suite (at least in Guzzle and OpenAI from what I glanced at).

My instinct tells me it's a namespace-related thing, nothing crazy.

Reply:
I will have a look to this. thanks!

Reply:
Fixed

Reply:
Perfect!